test/mocha/cli.js

```diff
@@ -55,7 +55,7 @@ describe("bin/uglifyjs", function () {
        exec(command, function (err, stdout) {
            if (err) throw err;
 
-           assert.strictEqual(stdout, "var bar=function(){function foo(bar){return bar}return foo}();\n" + 
+           assert.strictEqual(stdout, "var bar=function(){function foo(bar){return bar}return foo}();\n" +
```

Comment:
Sure, and yeah I've wondered why it changed. And yeah it was a space and luckily no other newline type.